### ISSUE TITLE
De-underline hyperlink on hover-out #2

### ISSUE THREAD

**Issue by PhMajerus** [microsoft/terminal#9141]
Testing Windows Terminal Preview 1.6.10412.0

De-underlining now works when the mouse moves away from the link while the window doesn't have the focus (#8925).
But we have a second de-underlining scenario that doesn't work properly...

Regardless of whether the window has the focus or not, hyperlinks are properly de-underlined only if the mouse moves over another character cell within the client area.
If the link is on the top row and the mouse moves higher than the link to the tabs/titlebar area, and/or if the link starts on the leftmost column and the mouse moves left to the margin and out of the window non-client area, or if the link reaches the rightmost column and the mouse moves right over the margin, scrollbar, etc..., the hyperlink stays hot.

Basically, it seems its hover-out handler is called only when the pointer moves over a cell not part of the hyperlink, but doesn't get called when the pointer moves directly to something else than the client area.

**Comment by DHowett** [comment]
More accurately, the hover handler is only called when you over over a thing that is the active terminal area. ;P

**Comment by DHowett** [comment]
you know, we should probably wire this up to a `PointerExited` handler in TermControl.

**Comment by Don-Vito** [comment]
> you know, we should probably wire this up to a `PointerExited` handler in TermControl.

Yep. Please assign it to me!
